apps/web/src/screens/Dashboard/Solana.tsx

```diff
@@ -26,6 +28,7 @@ interface Props {
 
 export const SolanaDashboard: FC<Props> = () => {
 	const [activeTabIndex, setActiveTabIndex] = useState(0);
+	const { _id, isPrivate } = useSnapshot(settingState);
```

Comment:
Maybe we need another name for `isPrivate`. When we have multiple settings, it causes confusion what need to private.